.github/workflows/start-gh-release.yml

```diff
@@ -8,10 +8,6 @@ on:
       - closed
 
 jobs:
-  check-for-moratorium:
-    uses: ./.github/workflows/tps-check-lock.yml
-    secrets: inherit
-
   get-source-branch-name:
     needs: [check-for-moratorium]
```

Comment:
Should we delete this `needs: [check-for-moratorium]` here?
```suggestion

```

Reply:
Good catch! Yes, thank you.